src/nvim/ex_cmds.c

```diff
@@ -73,6 +73,7 @@
  */
 typedef struct sign sign_T;
 
+static char_u e_pathtoolong[] = N_("E854: path too long for completion");
```

Comment:
Might as well move the one in `nvim/file_search.c` to `globals.h` (list starts at line 1086) and use that, instead of repeating it here.